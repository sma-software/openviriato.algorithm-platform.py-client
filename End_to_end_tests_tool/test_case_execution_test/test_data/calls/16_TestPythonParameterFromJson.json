{
    "restResource": "/trains/{trainID}/train-path-nodes",
    "httpMethod": "PUT",
    "urlParameters": [ "=shortTrain:ID=" ],
    "requestBody": [
        {
            "TrainPathNodeID": "=shortTrain:TrainPathNodes[0].ID=",
            "ArrivalTime": "2003-05-01T00:04:00",
            "DepartureTime": "2003-05-01T00:05:00"
        }
    ],
    "CSharpMethodName": "UpdateTrainTimes(trainID, times)",
    "CSharpMethodParameterTypes": [ "System.Int64", "IImmutableList{SMA.AlgorithmPlatform.AlgorithmInterface.AIDM.IUpdateTrainTimesNode}" ],
    "subcategory": "Update Train Times"
}
