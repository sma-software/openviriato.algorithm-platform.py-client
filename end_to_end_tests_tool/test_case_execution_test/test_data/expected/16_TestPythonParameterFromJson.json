{
  "ID": 2060,
  "DebugString": "RVZH_1_1_J03 tt_(G)",
  "TrainPathNodes":
  [
    {"ID": 1332,
      "ArrivalTime": "2003-05-01T00:04:00",
      "DepartureTime": "2003-05-01T00:05:00",
      "SectionTrackID": null,
      "NodeID": 18,
      "NodeTrackID": null,
      "FormationID": 1187,
      "MinimumRunTime": null,
      "MinimumStopTime": "P0D",
      "StopStatus": "CommercialStop",
      "SequenceNumber": 0
    },
    {
      "ID": 1696,
      "ArrivalTime": "2003-05-01T00:10:00",
      "DepartureTime": "2003-05-01T00:10:00",
      "SectionTrackID": 1171,
      "NodeID": 10,
      "NodeTrackID": null,
      "FormationID": null,
      "MinimumRunTime": "PT5M",
      "MinimumStopTime": "P0D",
      "StopStatus": "CommercialStop",
      "SequenceNumber": 1
    }
  ]
}